### PR TITLE
LPS-177494 Delete entries in the teardown method

### PR DESCRIPTION
**Pull request by adolfopa**
# What is this about?

https://issues.liferay.com/browse/LPS-177494

# How do I test it?

Follow the steps described in https://issues.liferay.com/browse/LPS-177494.

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

File: modules/apps/depot/depot-test/src/testIntegration/java/com/liferay/depot/staging/test/DepotEntryFileEntryFriendlyURLTest.java [hunk 1]
```diff
@@ -74,18 +73,21 @@ public void setUp() throws Exception {
...
+			Group stagingDepotEntryGroup = _stagingDepotEntry.getGroup();
+
+			if (stagingDepotEntryGroup.isStagedRemotely()) {
+				DepotStagingTestUtil.disableRemoteStaging(_stagingDepotEntry);
+			}
+			else {
+				DepotStagingTestUtil.disableStaging(_liveDepotEntry.getGroup());
+			}
+
+			_depotEntryLocalService.deleteDepotEntry(
```
[the remark this answers is not in the document]
Comment (adolfopa) [reply modules/apps/depot/depot-test/src/testIntegration/java/com/liferay/depot/staging/test/DepotEntryFileEntryFriendlyURLTest.java]: Applied you suggestions.

**Comment by adolfopa** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**